scripts/spack/packages/opensubdiv/package.py

```diff
@@ -28,15 +28,15 @@ def url_for_version(self, version):
 
     variant('tbb', default=False, description='Builds with Intel TBB support')
     variant('openmp', default=False, description='Builds with OpenMP support')
-    variant('doc', default=False, description='Builds documentation. Requires Python 2')
+    #variant('doc', default=False, description='Builds documentation. Requires Python 2')
     variant('pic', default=True,
             description='Produce position-independent code (for shared libs)')
 
     depends_on('cmake@2.8.6:', type='build')
-    depends_on('graphviz', type='build', when='+doc')
-    depends_on('doxygen', type='build', when='+doc')
-    depends_on('py-docutils', type='build', when='+doc')
-    depends_on('python@2.6:2', type='build', when='+doc')
+    # depends_on('graphviz', type='build', when='+doc')
+    # depends_on('doxygen', type='build', when='+doc')
+    # depends_on('py-docutils', type='build', when='+doc')
+    # depends_on('python@2.6:2', type='build', when='+doc')
```

Comment:
Is this a change from the latest package file, or do we need to set `~doc` on our opensubdiv dependency?

Reply:
This is from the latest package file.